### PR TITLE
fix: use customFetch instead of fetch

### PR DESCRIPTION
**Pull request by wa0x6e**
Closes https://github.com/snapshot-labs/workflow/issues/236

We already have an existing helper fetch function, supporting timeout

This PR replaces all direct calls to `fetch` from the cross-fetch lib, to our custom fetch helper, to take advantage of the timeout feature.

Now, all fetch calls from score-api have a max timeout of 20s

### CODE REVIEW

**Review by bonustrack — APPROVED** [review]
utACK

File: src/strategies/safe-vested/index.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import fetch from 'cross-fetch';
```
Comment (ChaituVR) [inline src/strategies/safe-vested/index.ts:1]: But we use fetch here https://github.com/snapshot-labs/snapshot-strategies/blob/eca18b39bf5093087ee6fdf98a6349dc882fdb8b/src/strategies/safe-vested/index.ts#L45

Comment (ChaituVR) [reply src/strategies/safe-vested/index.ts:1]: i will update it to use customFetch, but JIC if not needed LMK

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**